frontend/src/Home.test.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
Were these files copied and pasted over?

I think you might need to sync up to the current master and have the updated code :)